packages/ods/src/style/_modal.scss

```diff
@@ -48,6 +52,14 @@ $ods-modal-header-height: 32px;
     // We're not using variables because of a Chromium issue with the backdrops
     // See https://stackoverflow.com/questions/58818299/css-variables-not-working-in-dialogbackdrop/63322762#63322762
   }
+
+  @media screen and (width <= (breakpoint.$ods-breakpoint-mobile)) {
+    margin: auto 16px;
+    width: 100%;
+    min-width: fit-content;
+    max-width: fit-content;
+    max-height: revert;
```

Comment:
?

Reply:
BTW it is often simplier to set the small CSS as default and override as the breakpoint grows